### PR TITLE
Add account manager

### PR DESCRIPTION
**Pull request by tulimaki**
* Add PersonContact choice widget
* Add account manager for contact

### CODE REVIEW

**Comment by tulimaki** [comment]
retest this please

**Comment by tulimaki** [comment]
retest this please

File: doc/changelog.rst [hunk 1]
```diff
@@ -73,6 +73,8 @@ Localization
 Admin
 ~~~~~
 
+- Add account manger for contact
```
Comment (Pikkupomo) [inline doc/changelog.rst]: small typo here

File: shoop/admin/modules/contacts/views/list.py [hunk 2]
```diff
@@ -36,7 +36,12 @@ class ContactListView(PicotableListView):
     ]
 
     def get_queryset(self):
-        return super(ContactListView, self).get_queryset().annotate(n_orders=Count("customer_orders"))
+        filter = self.get_filter()
+        query = Q()
+        groups = filter.get("groups")
+        if groups:
+            query = Q(groups__in=groups)
+        return super(ContactListView, self).get_queryset().filter(query).annotate(n_orders=Count("customer_orders"))
```
Comment (Pikkupomo) [inline shoop/admin/modules/contacts/views/list.py]: This seems a bit mess, how about 
``` 
def get_queryset(self):
 	groups = self.get_filter().get("groups")
	query = Q(groups__in=groups) if groups else Q()
	return super(ContactListView, self).get_queryset().filter(query).annotate(n_orders=Count("customer_orders"))```

**Pull request merged**